shielder/contract/errors.rs

```diff
@@ -0,0 +1,19 @@
+use crate::traits::psp22::PSP22Error;
+
+#[derive(PartialEq, Debug, scale::Encode, scale::Decode)]
+#[cfg_attr(feature = "std", derive(scale_info::TypeInfo))]
```

Comment:
please use `#[ink::scale_derive(Encode, Decode, TypeInfo)]`; that way you don't need `scale` and `scale-info` dependencies